File: src/main/java/me/leodev/trollingfreedomreborn/trolls/Beds/BedMissing.java
package me.leodev.trollingfreedomreborn.trolls.Beds;

import org.bukkit.entity.Player;
import org.bukkit.event.Listener;

public class BedMissing implements Listener {

    public void BedMissing(Player player) {
        player.sendMessage("You have no home bed or charged respawn anchor, or it was obstructed");
    }
}

File: src/main/java/com/leomadrassi/trollingfreedomreborn/trolls/Beds/BedMissing.kt
package com.leomadrassi.trollingfreedomreborn.trolls.Beds

import org.bukkit.entity.Player
import org.bukkit.event.Listener

class BedMissing : Listener {
    fun BedMissing(player: Player) {
        player.sendMessage("You have no home bed or charged respawn anchor, or it was obstructed")
    }
}
